chapter_assets/email-templates/email-templates.md

```diff
@@ -0,0 +1,210 @@
+#Email templates
+
+## New Sponsor Information
+Hi XXX,
+
+I'm writing you to give you more detailed information about codebar.io and provide you with details for hosting a workshop.
+
+**About codebar**
+
+Codebar.io is an organization that started in the UK. The aim is to help women and other underrepresented groups in tech learn to program.
+
+We do this by pairing students with volunteer developers who work at different companies throughout Barcelona. We started a chapter in Barcelona over two years ago, and host workshops every two weeks. https://www.meetup.com/Codebar-Barcelona/
```

Comment:
I am not sure if meetup should be our reference for past workshop. what do you think?